test-cargo-miri/tests/test.rs

```diff
@@ -49,14 +49,26 @@ fn cargo_env() {
 }
 
 #[test]
-#[should_panic(expected="Explicit panic")]
-fn do_panic() { // In large, friendly letters :)
+#[should_panic(expected = "Explicit panic")]
+fn do_panic() {
+    // In large, friendly letters :)
```

Comment:
This comment moved around in incorrect ways (it commented the fn name, not the fn body)

Reply:
`rustfmt` frowned upon this file and reformated a bunch of stuff. I moved the comment so that it still comments the name, but doesn't annoy `rustfmt`.